src/server/index.ts

```diff
@@ -884,6 +884,10 @@ export class DocumentDriveServer extends BaseDocumentDriveServer {
         return this.addDriveOperations(drive, [operation]);
     }
 
+    async clearStorage() {
+        await this.storage.clearStorage?.();
```

Comment:
@acaldas Although this approach works in most of the cases, it doesn't guarantee that all the storage info is going to be cleared.
If for some reason you have drives that are not being tracked (after installing a new version of connect, incompatible drives... who knows 🤷🏻‍♂️), those drives info is going to be persisted in the storage, which is exactly what we want to avoid with this feature.

Reply:
@acaldas added the suggested change here: [7d8eb76](https://github.com/powerhouse-inc/document-drive/pull/82/commits/7d8eb76de37e8f5b6d5ed4eacbb2ee7121908643)